src/sentry/models/release.py

```diff
@@ -74,6 +98,11 @@ class Release(Model):
     projects = models.ManyToManyField(
         "sentry.Project", related_name="releases", through=ReleaseProject
     )
+    status = BoundedPositiveIntegerField(
```

Comment:
Having this in the model definition will cause a problem when this change is deployed as Django will expect the field to be present but it won't be until the migration is run which will be some time later.

If you omit the model field and use `migrations.SeparateDatabaseAndState` you'll be able to deploy the column migration,and then deploy the model change with its state change.

Reply:
That probably means making this column nullable? Otherwise inserts will fail. I think.

Reply:
Read operations are also a problem. Django will attempt to select this column when reading release records, and that will fail when the column is missing.

Reply:
Yeah, this column should be nullable by default. And since this `is_dangerous`, either this migration would have to be run before you deploy the rest of this pr, or you'd have to split out the table creation into a separate pr and merge/run that first.

Reply:
This already is the split out PR so I made this the one that only runs the SQL.